lib/SILGen/SILGen.cpp

```diff
@@ -483,6 +483,10 @@ static bool hasSILBody(FuncDecl *fd) {
   return fd->getBody(/*canSynthesize=*/false);
 }
 
+static bool haveProfiledAssociatedFuncDecl(SILDeclRef constant) {
+  return constant.isDefaultArgGenerator() || constant.isForeign;
```

Comment:
What about stored property initializers?

Reply:
I'm not sure. Isn't the decl associated with a stored property initializer some vardecl? If so, we wouldn't be at risk of instrumenting it twice.